Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/data/model/Tasks.java

```diff
@@ -85,4 +94,17 @@ public boolean apply(Map.Entry<Id, SyncedData<Task>> input) {
             }
         };
     }
+
+    public boolean hasSyncError() {
+        return !internalMap().filter(syncError()).isEmpty();
+    }
+
+    private static Predicate<Map.Entry<Id, SyncedData<Task>>> syncError() {
```

Comment:
maybe we can name this `onlySyncErrors` to make more clear what's the filter doing